lib/server.js

```diff
@@ -1,4 +1,4 @@
-const http = require('http');
+const https = require('https');
```

Comment:
It looks like out-of-scope changes.

Reply:
Yeah, will probably remove it but terraform provider registry requires HTTPS - once I've sorted the tests out, I should be able to remove https and https-localhost

Reply:
I have now removed `https-localhost` dependency as I added tests that utilise `ngrok`